examples/crm-full-stack/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We are adding the MIT license to the overall repository (it was always meant to be open source, it just got missed) in https://github.com/stripe/stripe-apps/pull/740. As such, please go ahead and remove the license file from your PR. Adding individual sub-licenses will get messy and complicated.